README.md

```diff
@@ -1,151 +1,149 @@
-coq-hott                 8.13        The Homotopy Type Theory library
-coq-interval             4.1.1       A Coq tactic for proving bounds on real-valued expressions automatically
-coq-mathcomp-algebra     1.12.0      Mathematical Components Library on Algebra
-coq-mathcomp-bigenough   1.0.0       A small library to do epsilon - N reasonning
-coq-mathcomp-character   1.12.0      Mathematical Components Library on character theory
-coq-mathcomp-field       1.12.0      Mathematical Components Library on Fields
-coq-mathcomp-fingroup    1.12.0      Mathematical Components Library on finite groups
-coq-mathcomp-finmap      1.5.1       Finite sets, finite maps, finitely supported functions
-coq-mathcomp-real-closed 1.1.2       Mathematical Components Library on real closed fields
-coq-mathcomp-solvable    1.12.0      Mathematical Components Library on finite groups (II)
-coq-mathcomp-ssreflect   1.12.0      Small Scale Reflection
-coq-menhirlib            20200624    A support library for verified Coq parsers produced by Menhir
-coq-mtac2                1.4+8.13    Mtac2: Typed Tactics for Coq
-coq-quickchick           1.5.0       Randomized Property-Based Testing Plugin for Coq
-coq-simple-io            1.5.0       IO monad for Coq
-coq-unicoq               1.5+8.13    An enhanced unification algorithm for Coq
-coq-vst                  2.7.1       Verified Software Toolchain
-gappa                    1.3.5       Tool intended for formally proving properties on numerical programs dealing with floating-point or fixed-point arithmetic
-menhir                   20200624    An LR(1) parser generator
-```
-- one unified setup script for Windows, macOS and Linux with few OS dependent sections only
-- on Windows there is an additional wrapper batch script to setup Cygwin as build and working environment
-- on Windows there is in addition a classic Windows installer mostly intended for quick installation by beginners
-- it is easy to build variants of the Windows installer with modified content from an existing opam switch
-- system prerequisites are installed using opam depext in a system independent manner
-- the script should be fairly robust and safe - it will immediately abort an all errors not explicitly handled
-- the script can be restarted if it fails - e.g cause of internet issues - it will not redo things it already did
-- the scripts are modular and easy to configure for specific needs, e.g. a reduced or extended setup for a lecture - see [`variants.md`](/variants.md)
+Please refer to the README file for your OS.
+
+- macOS: see [README_macOS](doc/README_macOS.md).
+- Windows: see [README_Windows](doc/README_Windows.md)
+- Linux: see [README_Linux](doc/README_Linux.md).
+
+## Additional information
+
+<details><summary><font size="+1">Licenses</font></summary>
+
+The Coq Platform setup scripts and the selection of packages and patches are licensed Creative Commons CC0.
```

Comment:
For clarity on what is meant by package in this specific sentence (not the source code of the package):

```suggestion
The Coq Platform setup scripts and the selection of package recipes and patches are licensed Creative Commons CC0.
```